sync-core/src/main/java/com/cloudant/mazha/CouchURIHelper.java

```diff
@@ -61,6 +63,17 @@ public URI changesUri(Map<String, Object> query) {
                 this.rootUriString,
                 "_changes"
         );
+
+        //lets find the since parameter
+        if(query.containsKey("since")){
+            Object since = query.get("since");
+            if(!(since instanceof String)){
```

Comment:
Yeah, I ran a test against couch 1.6 and it was failing because it was encoding some quotes in the url.